typesafe_conductr_cli/shazar.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
```

Comment:
If it doesn't hurt, let's keep it, because this way we can easily execute the script for testing purposes.